tests/functional/samples/test_samples_linux.py

```diff
@@ -349,6 +355,7 @@ def test_classification_async_cpp_gpu(self, tester, image, install_openvino_depe
         )
 
     @pytest.mark.vpu
+    @pytest.mark.xfail(reason='Network problems when downloading alexnet files')
```

Comment:
I meant parse logs and in case alexnet is faled - set xfail - because your current xfail may lost a real problem